plugins/woocommerce/src/Blocks/BlockTypes/Checkout.php

```diff
@@ -242,6 +244,71 @@ protected function is_checkout_endpoint() {
 		return is_wc_endpoint_url( 'order-pay' ) || is_wc_endpoint_url( 'order-received' );
 	}
 
+	/**
+	 * Update the local pickup title in WooCommerce Settings when the checkout page containing a Checkout block is saved.
+	 *
+	 * @param int      $post_id The post ID.
+	 * @param \WP_Post $post    The post object.
+	 * @return void
+	 */
+	public function update_local_pickup_title( $post_id, $post ) {
+		if ( empty( $post->post_status ) || 'inherit' === $post->post_status ) {
+			// This is not a proper save action, maybe an autosave, so don't continue.
+			return;
+		}
```

Comment:
What are your thoughts on moving the comment above the conditional check?

```suggestion
		// This is not a proper save action, maybe an autosave, so don't continue.
		if ( empty( $post->post_status ) || 'inherit' === $post->post_status ) {
			return;
		}
```